docs/byoh-instance-pre-requisites.md

```diff
@@ -0,0 +1,11 @@
+# BYOH Instance Pre-requisites
+
+The following pre-requisites must be fulfilled in order to add a Windows BYOH node.
+* The Docker container runtime must be installed on the instance.
+* The instance must be reachable by the OpenShift cluster.
+* Port 22 must be open and running [an SSH server](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
+* An administrator user is present with the [provided private key](/README.md#create-a-private-key-secret) set as an authorized SSH key.
```

Comment:
`provided` seems strange. I suggest explicitly saying the private key used in the secret.